### ISSUE TITLE
How to edit the model list???

### ISSUE THREAD

**Issue by DocMAX** [h2oai/h2ogpt#1161]
![grafik](https://github.com/h2oai/h2ogpt/assets/5351323/95f1a8c6-ed76-4ebb-b160-3f1a2257475e)

How to edit the model list???

**Comment by pseudotensor** [comment]
https://github.com/h2oai/h2ogpt/assets/2249614/dfebd7dd-4d95-4390-b514-ff9b76af9664

**Comment by pseudotensor** [comment]
Also, if you login, then which models you added and loaded will be added to the list from then on.

**Comment by pseudotensor** [comment]
@DocMAX I've re-worked the UI hundreds of times :)  I'm not a UI guy.  I'm limited by what gradio can do.  For that particular element, why I say in the description "CHOOSE/ENTER"

**Comment by DocMAX** [comment]
I would like to suggest a dynamicly filled list. Based on files that are in /workspace/dir/ this this should show any files available. Not predefined static entries.

**Comment by pseudotensor** [comment]
Yes, I've added a specific path passed at CLI time for llama.cpp models.

Yes, and I've added this specific dynamic-filled capability, at least at startup time.

One would also like to be able to see meta data, like whether on disk or just "curated" ones, which prompt_type would be used, size of model, etc.  Can add that later once...

One would also like to be able to explorer the disk, but that will have to wait for gradio team to fix various gradio4 bugs:  https://github.com/h2oai/h2ogpt/issues/1166
